test/algorithms/afl/loop.cpp

```diff
@@ -63,7 +63,7 @@ static void AFLLoop(bool forksrv) {
   std::shared_ptr<AFLSetting> setting(new AFLSetting(
       {"../../put_binaries/libjpeg/libjpeg_turbo_fuzzer", "@@"},
       input_dir.native(), output_dir.native(), option::GetExecTimeout<AFLTag>(),
-      option::GetMemLimit<AFLTag>(), forksrv, false, /* dumb_mode*/
+      option::GetMemLimit<AFLTag>(true), forksrv, false, /* dumb_mode*/
```

Comment:
I think we don't need to stick to the previous value.
```suggestion
      option::GetMemLimit<AFLTag>(), forksrv, false, /* dumb_mode*/
```